### PR TITLE
update the Dockerfile

### PR DESCRIPTION
**Pull request by gkwan-ibm**
`open-liberty:springBoot2` won't be updated with new fixpacks anymore, so need to use `open-liberty` image.

### CODE REVIEW

**Comment by anjumfatima90** [comment]
Yes, server.xml is needed to add the features necessary to run the spring boot application. The Dockerfile of `open-liberty:springBoot2` had the step `RUN cp /opt/ol/wlp/templates/servers/springBoot2/server.xml /config/server.xml`. Since we are using the `open-liberty` image, we would have to do it in the application's Dockerfile.

**Comment by gkwan-ibm** [comment]
Cannot make a soft-link directly because /config/server.xml exists on the `open-liberty` image. Also, running `ln` need to run `USER root` and `USER 1001` after.

**Comment by barecode** [comment]
I lament the loss of a simpler Dockerfile. However, I understand why the change is occuring. One suggested wording improvement then I'll be content.

File: README.adoc [hunk 1]
```diff
@@ -96,9 +96,7 @@ The first stage copies the [hotspot=copyJar]`guide-spring-boot-0.1.0.jar` Spring
...
 For more information about the `springBootUtility` command, see the https://www.ibm.com/support/knowledgecenter/en/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/rwlp_command_sbutility.html[springBootUtility documentation^].
 
-The second stage begins with the [hotspot=OLimage2 file=0]`Open Liberty Docker image`. Then, the Dockerfile copies 
-the Spring Boot dependent library JAR files that are at [hotspot=libcache file=0]`lib.index.cache` and
-the [hotspot=thinjar file=0]`thin-guide-spring-boot-0.1.0.jar` file, both of which were generated in the first stage.
+The second stage begins with the [hotspot=OLimage2 file=0]`Open Liberty Docker image`. The Dockerfile copies the [hotspot=serverXml file=0]`server.xml` file from the `/opt/ol/wlp/templates` directory. Then, the Dockerfile copies the Spring Boot dependent library JAR files that are at [hotspot=libcache file=0]`lib.index.cache` directory and the [hotspot=thinjar file=0]`thin-guide-spring-boot-0.1.0.jar` file. The `lib.index.cache` directory and the `thin-guide-spring-boot-0.1.0.jar` file were both generated in the first stage.
```
Comment (barecode) [inline README.adoc]: Can we explain a bit more about why this particular server.xml? Something along the lines of:

The Dockerfile copies the pre-configured Spring Boot [hotspot=serverXml file=0]`server.xml` file from the `/opt/ol/wlp/templates` directory, which enables Spring Boot and TLS support.

Comment (gkwan-ibm) [reply README.adoc]: updated.

@Charlotte-Holt are you OK for the change?

Comment (Charlotte-Holt) [reply README.adoc]: @gkwan-ibm Yes, looks good. Thanks.

**Pull request merged**